src/Prelude/Linear/Internal.hs

```diff
@@ -19,13 +19,13 @@ import Data.Functor.Identity
 
 -- | Beware: @($)@ is not compatible with the standard one because it is
 -- higher-order and we don't have multiplicity polymorphism yet.
-($) :: (a %1-> b) %1-> a %1-> b
+($) :: (a %p-> b) %1-> a %p-> b
 -- XXX: Temporary as `($)` should get its typing rule directly from the type
 -- inference mechanism.
```

Comment:
You can remove the comment per https://github.com/ghc-proposals/ghc-proposals/pull/406